api-reference/10 UI Components/BaseChart/1 Configuration/tooltip/customizeTooltip.md

```diff
@@ -197,6 +197,110 @@ The format of the string is the following: "h: %highValueText% o: %openValueText
     </tr>
 </table>
 
```

Comment:
Can you apply the following changes above?

Line 45:  
What do you mean by "the similar type"? Is the type _the same_? For example:  
"In this instance, <b>argument</b> has the same type as the argument axis."

Line 69:  
"folllowing" is misspelled. Please fix.

Line 82:  
<td>The percent value of the point being hovered over.

This description is exactly the same as the one on line 78. Is this intended?  
Also note that the closing td tag is missing.

Line 101:  
Please update this description to the following:  
"The first range value of the point being hovered over as it is set in the data source."

Line 113:  
Please update this to the following:  
"The second range value of the point being hovered over as it is set in the data source."

Reply:
Done